engine/components-api/src/main/java/com/cloud/resource/ResourceManager.java

```diff
@@ -61,6 +61,11 @@ public interface ResourceManager extends ResourceService, Configurable {
                     + "To force-stop VMs, choose 'ForceStop' strategy",
             true, ConfigKey.Scope.Global, null, null, null, null, null, ConfigKey.Kind.Select, "Error,Migration,ForceStop");
 
+    ConfigKey<String> SystemVMDefaultHypervisor = new ConfigKey<String>(String.class,
+            "system.vm.default.hypervisor", "Advanced", "Any", "Hypervisor type used to create System VMs. Valid values are: XenServer, KVM, VMware, Hyperv, VirtualBox, " +
+            "Parralels, BareMetal, Ovm, LXC, Any", true, ConfigKey.Scope.Global, null, null, null, null, null, ConfigKey.Kind.Select, "XenServer, KVM, VMware, Hyperv, " +
+            "VirtualBox, Parralels, BareMetal, Ovm, LXC, Any");
```

Comment:
@sureshanaparti I agree with changing the scope to zone, however, can you further explain what you mean by handling "defaultSystemVMHypervisorType" in upgrade?

Wouldn't changing the scope to zone and fetching the value of the config for the specific zone be sufficient to guarantee the expected behaviour?

Reply:
@GaOrtiga 
should the default value be "None" ?

Reply:
@weizhouapache 
Yes, the default must be "None" to keep the current behaviour. However, since this configuration is only setting a default, "None" and "Any" have the same behaviour.

Reply:
@GaOrtiga 
I quickly checked the code. it looks like the default value is NULL which is same as "None"
I suggest we keep it empty or None.